### DISCUSSION TITLE
[feature request] Add gallery view​

### DISCUSSION THREAD

**Discussion by EnterMan123** [toeverything/blocksuite#6581] [General]
### Description

Hope to add gallery view, just like notion 🙏

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by EYHN** [comment]
Are you referring to the Gallery view of the database?

like notion? https://www.notion.so/help/galleries

**Comment by EnterMan123** [comment]
> Are you referring to the Gallery view of the database?
> 
> like notion? https://www.notion.so/help/galleries

Yes, I think it's a great feature and it would be great if we could see it in affine too.

**Reply by hwangdev97** [reply]
We do have plans for a gallery view, but we want to release the views of the database and all documents together. So this will require some time for design and development.

![image](https://github.com/toeverything/blocksuite/assets/24713927/443a8556-756e-457c-b8ef-a1c0ea64f82a)

**Comment by taylerodea** [comment]
I wonder in 8 months this topic moved @hwangdev97

**Reply by hwangdev97** [reply]
I'm sorry 🥲 , this feature was traded off during the iteration. It will first appear in another form on the mobile. 

We may provide a similar layout after the mobile end goes live.
![IMG_7396FCD51891-1](https://github.com/user-attachments/assets/a21e32db-7222-46ee-a173-e2d88d07d209)
